packages/zostso/src/index.ts

```diff
@@ -39,6 +39,9 @@ export * from "./IssueTso";
 export * from "./PingTso";
 export * from "./SendTso";
 export * from "./StartTso";
+export * from "./StartTsoApp";
+export * from "./SendTsoApp";
+export * from "./ReceiveTsoApp";
```

Comment:
Not sure if it's too much trouble, but seems like some porperties are shared between these new interfaces 😋 

Curious if you considered combining them (and inheriting through extension, i.e. using `extends`)